src/examples/pane/PaneHorizontalCollapse.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { PaneProvider, Pane } from '@zendeskgarden/react-grid';
+import useResizeObserver from 'use-resize-observer';
+import { XL, Paragraph } from '@zendeskgarden/react-typography';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import styled from 'styled-components';
+
+const StyledDiv = styled.div`
+  padding: ${p => p.theme.space.base * 6}px ${p => p.theme.space.base * 6}px
+    ${p => p.theme.space.base * 4}px;
+  height: 100%;
+`;
+
+const StyledParagraph = styled(Paragraph)`
+  margin-top: ${p => p.theme.space.xs};
+`;
+
+const Example = () => {
+  const { ref, width = 1, height = 1 } = useResizeObserver();
+
+  return (
+    <div ref={ref}>
+      <PaneProvider
+        totalPanesHeight={height}
+        totalPanesWidth={width}
+        defaultRowValues={{
+          'row-1': 4,
+          'row-2': 4
+        }}
+      >
+        {({ getGridTemplateRows }) => (
+          <div
+            style={{
+              width: '100%',
+              height: '250px',
+              display: 'grid',
+              gridTemplateRows: getGridTemplateRows()
+            }}
+          >
+            <Pane>
+              <StyledDiv>
+                <Pane.Content>
+                  <XL>Pane 1</XL>
+                  <StyledParagraph>
+                    Turnip greens yarrow ricebean rutabaga endive cauliflower sea lettuce kohlrabi
+                    amaranth water spinach avocado.
+                  </StyledParagraph>
+                </Pane.Content>
+              </StyledDiv>
+            </Pane>
+            <Pane
+              style={{
+                backgroundColor: PALETTE.grey[100]
+              }}
```

Comment:
[nit] ditto.

Reply:
Updated in https://github.com/zendeskgarden/website/pull/416/commits/f00aab8b4acb4eb8cd3dc995da1153e5fc5c10a3.